integration/custom_deploy/Tiltfile

```diff
@@ -0,0 +1,26 @@
+
+v1alpha1.file_watch(
+    name='custom-deploy-fw',
+    watched_paths=['./deploy.yaml']
+)
+
+v1alpha1.kubernetes_apply(
```

Comment:
nice!

Reply:
It was really fast/enjoyable to use these helpers! I didn't really refer to docs and the only mistake I made was originally passing a number instead of duration string for `Timeout`